addons/mail/models/mail_link_preview.py

```diff
@@ -63,35 +63,25 @@ def _create_from_message_and_notify(self, message):
     def _hide_and_notify(self):
         if not self:
             return True
-        notifications = [
-            (
-                link_preview.message_id._bus_notification_target(),
-                'mail.record/insert', {
-                    'Message': {
-                        'linkPreviews': [('DELETE', {'id': link_preview.id})],
-                        'id': link_preview.message_id.id,
-                    }
+        for mail_link_preview in self:
+            mail_link_preview.message_id._bus_notification_target()._bus_send('mail.record/insert', {
```

Comment:
`_bus_channel` on link preview too?

Reply:
I didn't do it because the message is implementing it already.
Should I?

Reply:
It's a minor model/detail so both options are fine.